### PR TITLE
add price functions

### PR DESCRIPTION
**Pull request by ralph-pichler**

### CODE REVIEW

File: src/PriceOracle.sol [hunk 1]
```diff
@@ -2,4 +2,22 @@
 pragma solidity ^0.8.4;
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract PriceOracle is Ownable {}
+contract PriceOracle is Ownable {
+    uint256 public price;
```
Comment (Eknir) [inline src/PriceOracle.sol:21]: Comment, including the units

File: src/PriceOracle.sol [hunk 2]
```diff
@@ -2,4 +2,22 @@
...
+    uint256 public chequeValueDeduction;
+
+    function getPrice() external view returns (uint256, uint256) {
```
Comment (Eknir) [inline src/PriceOracle.sol:33]: Natspec comments in functions and contract

File: src/PriceOracle.sol (hunk 1 above)
Comment (ralph-pichler) [reply src/PriceOracle.sol:21]: done

File: src/PriceOracle.sol (hunk 2 above)
Comment (ralph-pichler) [reply src/PriceOracle.sol:33]: done

File: src/PriceOracle.sol [hunk 3]
```diff
@@ -2,4 +2,51 @@
...
+     */
+    event PriceUpdate(uint256 price);
+    /**
+     * @dev Emitted when the cheque value deduction amount is updated.
+     */
+    event ChequeValueDeductionUpdate(uint256 chequeValueDeduction);
+
+    // current price in PLUR per accounting unit
+    uint256 public price;
+    // value deducted from first received cheque in PLUR per accounting unit
```
Comment (Eknir) [inline src/PriceOracle.sol]: first received cheque from a peer

File: src/PriceOracle.sol [hunk 4]
```diff
@@ -2,4 +2,51 @@
...
+    uint256 public chequeValueDeduction;
+
+    /**
+     * @notice Returns the current price in PLUR per accounting unit
```
Comment (Eknir) [inline src/PriceOracle.sol]: comment is not accurate

File: src/PriceOracle.sol [hunk 5]
```diff
@@ -2,4 +2,51 @@
...
+     */
+    event PriceUpdate(uint256 price);
+    /**
+     * @dev Emitted when the cheque value deduction amount is updated.
+     */
+    event ChequeValueDeductionUpdate(uint256 chequeValueDeduction);
+
+    // current price in PLUR per accounting unit
+    uint256 public price;
+    // value deducted from first received cheque from a peer in PLUR per accounting unit
```
Comment (Eknir) [inline src/PriceOracle.sol]: add unit

**Comment by Eknir** [comment]
Can you please include a README, instructing how to run the tests?

**Comment by Eknir** [comment]
Could you add a production-ready deployment script (can in follow-up PR), where the script assigns ownership of the price oracle to another (configurable) address after deployment is complete?

**Review by Eknir — APPROVED** [review]
LGTM!

**Pull request merged**